compiler/typecheck/TcRnTypes.hs

```diff
@@ -666,6 +667,10 @@ data TcGblEnv
         tcg_fords     :: [LForeignDecl GhcTc], -- ...Foreign import & exports
         tcg_patsyns   :: [PatSyn],            -- ...Pattern synonyms
 
+        tcg_daml_templates :: [Name],  -- ...Daml templates
+        tcg_daml_interfaces :: [Name], -- ...Daml interfaces
+        tcg_daml_choices :: [Name],    -- ...Daml choices
```

Comment:
I couldn't find these anywhere else (other than `compiler/typecheck/TcRnMonad.hs`, where they're just set to `[]`), did I miss something?

Reply:
The tcg_daml_{templates,interfaces,choices} fields were just a holdover from a previous iteration - I've removed them as they are now deprecated.